cpp/src/parquet/arrow/reader.h

```diff
@@ -115,17 +115,32 @@ class RowGroupReader;
 // arrays
 class PARQUET_EXPORT FileReader {
  public:
-  /// Factory function to create a FileReader from a ParquetFileReader and properties
+  // Factory function to create a FileReader from a ParquetFileReader and properties.
+  // \deprecated Deprecated in 19.0.0. Use arrow::Result version instead.
+  ARROW_DEPRECATED("Deprecated in 19.0.0. Use arrow::Result version instead.")
   static ::arrow::Status Make(::arrow::MemoryPool* pool,
                               std::unique_ptr<ParquetFileReader> reader,
                               const ArrowReaderProperties& properties,
                               std::unique_ptr<FileReader>* out);
 
-  /// Factory function to create a FileReader from a ParquetFileReader
+  // Factory function to create a FileReader from a ParquetFileReader and properties
+  // Returns an arrow::Result containing a unique pointer to the FileReader.
```

Comment:
Could you use `\return` Doxygen markup?
https://www.doxygen.nl/manual/commands.html#cmdreturn